docs/sphinx/Physics/DiskModel.md

```diff
@@ -0,0 +1,323 @@
+3. Particle-Self-Gravity
+  - uses the self-gravity module to effectively achieve a hybrid between the other 2 modes.
+  - enable this mode by setting the `gravity.gas_only_use_static_grav` runtime parameter to `true`
+  - The gravitational forces are handled differently for star particles and gas:
+    - gravity source terms for the gas are only computed from the static potentials: {math}`(\Phi_{\rm dm} + \Phi_{\rm stars,old})`.
+    - the star particles experience forces from all components {math}`(\Phi_{\rm dm} + \Phi_{\rm stars,old} + \Phi_{\rm stars,young} + \Phi_{\rm gas})`.
+    - This merits some emphasis: star particles do indeed experience gravity from the gas (as well as from themselves) while the gas does **not** experience its own self-gravity. This choice is motivated by the assumptions we make for the gravity-solver to compute contributions from self-gravity [down below](#analytic-self-gravity-estimate)
+  - **Purpose:** this is intended to be an intermediate point between both of the other modes that lets us confidently experiment with a 3-phase ISM without self-consistent star formation.
+
+
+
+
+
+
+
+## Analytic Gravitational Potentials
+
+In this section, we discuss the analytic formulae used for modeling Gravitational Potentials. It may be instructive to look back at our [definitions of various components](#primer-on-gravitational-potentials).
+### Static Potentials
+
+First, let's consider static potentials. 
+
+All configurations of the idealized Galaxy Simulations:
+- currently assume that you are using a static Gravitational Potential produced by 2 components: (i) component contributed by older stellar populations {math}`\Phi_{\rm stars-old}` and (ii) a component contributed by dark matter in the halo, {math}`\Phi_{\rm halo}`.
+- use these potentials for initializations and for evolving the simulation over a timestep.
+
+:::{important}
+[As we noted above](#configurations), when using the standalone Static-Gravity module, the logic implementing the static potential is duplicated between the initial conditions and the source-term calculations; it's the user's responsibility to ensure logic remains consistent in both parts of the code. 
+This is **NOT** a concern when using the Self-Gravity module
+:::
+
+Consider the parameterization of the potential of a Miyamoto-Nagai potential (this will be important [again](#analytic-self-gravity-estimate), shortly)
+
+:::{math}
+\Phi_{\rm MN}(R,z; M,a,b)\equiv \frac{-G M}{\sqrt{R^2 + (a + \sqrt{z^2 + b^2})^2}}.
+:::
+
+At the time of writing, we always define {math}`\Phi_{\rm stars,old}(R,z)=\Phi_{\rm MN}(R,z; M_{\rm stars}, R_{\rm stars}, z_{\rm stars})`, where {math}`R` & {math}`z` are cylindrical coordinates, {math}`R_{\rm stars}` & {math}`z_{\rm stars}` are the scale radius/height, and {math}`M_{\rm stars}` is the mass of the disk.
+
```

Comment:
Would it be possible in this section to give a brief description of how these parameters are set? I.e. are they hardcoded in the disk model, are they runtime parameters...?